### ISSUE TITLE
Not able to generate any good output using ConsisID

### ISSUE THREAD

**Issue by nitinmukesh** [huggingface/diffusers#10659]
### Describe the bug

I tried with 5-6 different image and prompts. Some do produce output but very bad.

https://github.com/user-attachments/assets/e9eccd72-a3fb-4ba0-86a3-04dc6f28a7f9

### Reproduction

tried with higher num_frames too.

```
import torch
from diffusers import ConsisIDPipeline
from diffusers.pipelines.consisid.consisid_utils import prepare_face_models, process_face_embeddings_infer
from diffusers.utils import export_to_video
from huggingface_hub import snapshot_download

snapshot_download(repo_id="BestWishYsh/ConsisID-preview", local_dir="BestWishYsh/ConsisID-preview")

face_helper_1, face_helper_2, face_clip_model, face_main_model, eva_transform_mean, eva_transform_std = prepare_face_models("BestWishYsh/ConsisID-preview", device="cuda", dtype=torch.bfloat16)

pipe = ConsisIDPipeline.from_pretrained("BestWishYsh/ConsisID-preview", torch_dtype=torch.bfloat16)
# pipe.to("cuda")
pipe.vae.enable_slicing()
pipe.vae.enable_tiling()
# pipe.enable_model_cpu_offload()
pipe.enable_sequential_cpu_offload()
prompt = "The video captures a young woman walking confidently along a lively urban street during golden hour, bathed in warm sunlight. The scene is rich with color, as neon signs flicker in the background, and the city buzzes with movement. Her long blonde hair sways gently as she walks, catching the light, while her soft expression and graceful stride give off a dynamic yet effortless charm. The modern cityscape behind her is alive with activity—passersby, glowing shopfronts, and distant cars adding depth to the scene. The crisp details of her outfit, the playful folds of her pleated skirt, and the subtle glint of her accessories enhance the vibrant and high-energy aesthetic. The lighting creates a cinematic glow, highlighting her features and adding a sense of motion and realism to the animation. The atmosphere is upbeat and engaging, evoking the feel of a trendy, youthful city adventure."
# image = "https://github.com/PKU-YuanGroup/ConsisID/blob/main/asserts/example_images/2.png?raw=true"
image="C:/Users/nitin/OneDrive/Desktop/8.jpeg"
id_cond, id_vit_hidden, image, face_kps = process_face_embeddings_infer(face_helper_1, face_clip_model, face_helper_2, eva_transform_mean, eva_transform_std, face_main_model, "cuda", torch.bfloat16, image, is_align_face=True)

video = pipe(
    image=image, 
    prompt=prompt,
    # negative_prompt=negative_prompt,
    width=720,
    height=480,
    num_frames=17,
    use_dynamic_cfg=False,
    num_inference_steps=50,
    guidance_scale=6.0,
    id_vit_hidden=id_vit_hidden, 
    id_cond=id_cond, 
    kps_cond=face_kps, 
    generator=torch.Generator("cuda").manual_seed(42)
)
export_to_video(video.frames[0], "output.mp4", fps=8)
```

![Image](https://github.com/user-attachments/assets/cba87432-f40f-45c1-90a1-3a2c30ac1d28)

### Logs

```shell
(venv) C:\aiOWN\diffuser_webui>python ConsisID.py
import error: No module named 'triton'
Please 'pip install xformers'
Optional: You can run 'pip install apex' to accelerate.
Optional: You can run 'pip install xformers' to accelerate.
Fetching 36 files: 100%|█████████████████████████████████████████| 36/36 [00:00<00:00, 503.39it/s]
C:\aiOWN\diffuser_webui\venv\lib\site-packages\torchvision\models\_utils.py:208: UserWarning: The parameter 'pretrained' is deprecated since 0.13 and may be removed in the future, please use 'weights' instead.
  warnings.warn(
C:\aiOWN\diffuser_webui\venv\lib\site-packages\torchvision\models\_utils.py:223: UserWarning: Arguments other than a weight enum or `None` for 'weights' are deprecated since 0.13 and may be removed in the future. The current behavior is equivalent to passing `weights=None`.
  warnings.warn(msg)
C:\aiOWN\diffuser_webui\venv\lib\site-packages\facexlib\detection\__init__.py:22: FutureWarning: You are using `torch.load` with `weights_only=False` (the current default value), which uses the default pickle module implicitly. It is possible to construct malicious pickle data which will execute arbitrary code during unpickling (See https://github.com/pytorch/pytorch/blob/main/SECURITY.md#untrusted-models for more details). In a future release, the default value for `weights_only` will be flipped to `True`. This limits the functions that could be executed during unpickling. Arbitrary objects will no longer be allowed to be loaded via this mode unless they are explicitly allowlisted by the user via `torch.serialization.add_safe_globals`. We recommend you start setting `weights_only=True` for any use case where you don't have full control of the loaded file. Please open an issue on GitHub for any issues related to this experimental feature.
  load_net = torch.load(model_path, map_location=lambda storage, loc: storage)
C:\aiOWN\diffuser_webui\venv\lib\site-packages\facexlib\parsing\__init__.py:20: FutureWarning: You are using `torch.load` with `weights_only=False` (the current default value), which uses the default pickle module implicitly. It is possible to construct malicious pickle data which will execute arbitrary code during unpickling (See https://github.com/pytorch/pytorch/blob/main/SECURITY.md#untrusted-models for more details). In a future release, the default value for `weights_only` will be flipped to `True`. This limits the functions that could be executed during unpickling. Arbitrary objects will no longer be allowed to be loaded via this mode unless they are explicitly allowlisted by the user via `torch.serialization.add_safe_globals`. We recommend you start setting `weights_only=True` for any use case where you don't have full control of the loaded file. Please open an issue on GitHub for any issues related to this experimental feature.
  load_net = torch.load(model_path, map_location=lambda storage, loc: storage)
C:\aiOWN\diffuser_webui\venv\lib\site-packages\onnxruntime\capi\onnxruntime_inference_collection.py:115: UserWarning: Specified provider 'CUDAExecutionProvider' is not in available provider names.Available providers: 'AzureExecutionProvider, CPUExecutionProvider'
  warnings.warn(
Applied providers: ['CPUExecutionProvider'], with options: {'CPUExecutionProvider': {}}
C:\aiOWN\diffuser_webui\venv\lib\site-packages\consisid_eva_clip\factory.py:94: FutureWarning: You are using `torch.load` with `weights_only=False` (the current default value), which uses the default pickle module implicitly. It is possible to construct malicious pickle data which will execute arbitrary code during unpickling (See https://github.com/pytorch/pytorch/blob/main/SECURITY.md#untrusted-models for more details). In a future release, the default value for `weights_only` will be flipped to `True`. This limits the functions that could be executed during unpickling. Arbitrary objects will no longer be allowed to be loaded via this mode unless they are explicitly allowlisted by the user via `torch.serialization.add_safe_globals`. We recommend you start setting `weights_only=True` for any use case where you don't have full control of the loaded file. Please open an issue on GitHub for any issues related to this experimental feature.
  checkpoint = torch.load(checkpoint_path, map_location=map_location)
Applied providers: ['CPUExecutionProvider'], with options: {'CPUExecutionProvider': {}}
find model: BestWishYsh/ConsisID-preview\face_encoder\models\antelopev2\1k3d68.onnx landmark_3d_68 ['None', 3, 192, 192] 0.0 1.0
Applied providers: ['CPUExecutionProvider'], with options: {'CPUExecutionProvider': {}}
find model: BestWishYsh/ConsisID-preview\face_encoder\models\antelopev2\2d106det.onnx landmark_2d_106 ['None', 3, 192, 192] 0.0 1.0
Applied providers: ['CPUExecutionProvider'], with options: {'CPUExecutionProvider': {}}
find model: BestWishYsh/ConsisID-preview\face_encoder\models\antelopev2\genderage.onnx genderage ['None', 3, 96, 96] 0.0 1.0
Applied providers: ['CPUExecutionProvider'], with options: {'CPUExecutionProvider': {}}
find model: BestWishYsh/ConsisID-preview\face_encoder\models\antelopev2\glintr100.onnx recognition ['None', 3, 112, 112] 127.5 127.5
Applied providers: ['CPUExecutionProvider'], with options: {'CPUExecutionProvider': {}}
find model: BestWishYsh/ConsisID-preview\face_encoder\models\antelopev2\scrfd_10g_bnkps.onnx detection [1, 3, '?', '?'] 127.5 128.0
set det-size: (640, 640)
Loading checkpoint shards: 100%|████████████████████████████████████| 2/2 [00:00<00:00,  5.97it/s]
Loading pipeline components...: 100%|███████████████████████████████| 5/5 [00:02<00:00,  2.31it/s]
100%|█████████████████████████████████████████████████████████████| 50/50 [12:11<00:00, 14.63s/it]
```

### System Info

- 🤗 Diffusers version: 0.33.0.dev0
- Platform: Windows-10-10.0.26100-SP0
- Running on Google Colab?: No
- Python version: 3.10.11
- PyTorch version (GPU?): 2.5.1+cu124 (True)
- Flax version (CPU?/GPU?/TPU?): not installed (NA)
- Jax version: not installed
- JaxLib version: not installed
- Huggingface_hub version: 0.27.1
- Transformers version: 4.48.1
- Accelerate version: 1.4.0.dev0
- PEFT version: not installed
- Bitsandbytes version: 0.45.0
- Safetensors version: 0.5.2
- xFormers version: not installed
- Accelerator: NVIDIA GeForce RTX 4060 Laptop GPU, 8188 MiB
- Using GPU in script?: <fill in>
- Using distributed or parallel set-up in script?: <fill in>

### Who can help?

@SHYuanBest

**Comment by SHYuanBest** [comment]
try `pip install triton`?

**Comment by a-r-r-o-w** [comment]
@nitinmukesh Could you try with 49 frames? CogVideoX 1.0 models were trained with fixed frames (49) and are known to perform worse with other frame values. Similarly, CogVideoX 1.5 perform best at 49 and 81 frames

**Comment by nitinmukesh** [comment]
@SHYuanBest 

Will install Triton and try again.

@a-r-r-o-w 
Thank you for this info. I did tried with 49 too earlier but poor results, Will try again now after installing Triton.

**Comment by SHYuanBest** [comment]
> [@SHYuanBest](https://github.com/SHYuanBest)
> 
> Will install Triton and try again.
> 
> [@a-r-r-o-w](https://github.com/a-r-r-o-w) Thank you for this info. I did tried with 49 too earlier but poor results, Will try again now after installing Triton.

You can try turn off these two parameters (which will result in a loss of performance reported [here](https://github.com/PKU-YuanGroup/ConsisID?tab=readme-ov-file#gpu-memory-optimization)).
```
pipe.vae.enable_slicing()
pipe.vae.enable_tiling()
```

**Comment by nitinmukesh** [comment]
I am fine with reduced quality as I have 8 GB VRAM only. Just want to produce decent output (can post process and improve) before I include this feature in WebUI.

**Comment by nitinmukesh** [comment]
Not sure why but I generated some good content using CogVideoX (t2v and i2v) with same settings. So if this model is based on Cog, should work similar.
https://youtu.be/mYzjy7tDdAM?t=54
https://github.com/user-attachments/assets/7397a6ea-808e-43bd-993c-ec52e9b4a5f2

Output
https://github.com/user-attachments/assets/42e753ab-36e0-466c-b40f-b4acf904c72e
